index.html

```diff
@@ -773,8 +824,10 @@ <h4>Creation</h4>
                                     and a Location header containing a system-generated identifier for the TD.
                                 </span>
                                 <span class="rfc2119-assertion" id="tdd-reg-create-anonymous-td-generated-id"></span>
-                                    The identifier SHOULD be a Version 4 UUID URN [[RFC4122]].
+                                    The identifier SHOULD be a UUID Version 4 [[RFC4122]].
                                 </span>
+                                That is a random or pseudo-random number which does not carry unintended information
+                                about the host or the resource.
```

Comment:
@mmccool should we add some recommendation about generation of secure identifiers in the privacy and security section? In particular, UUID versions other than v4 may need to be avoided.

Reply:
please create an issue about this in wot-security and we will discuss